pkg/serverless/tags/tags.go

```diff
@@ -134,6 +134,11 @@ func AddColdStartTag(tags []string, coldStart bool) []string {
 	return tags
 }
 
+// GetExtensionVersion returns the extension version which is fed at build time
+func GetExtensionVersion() string {
```

Comment:
Why do we need a function for this? Can we just use the variable/constant directly?